bootstrap-salt.sh

```diff
@@ -4810,6 +4836,7 @@ install_opensuse_stable_deps() {
     fi
 
     # Is the repository already known
+    # FIXME: cleartext download over unsecure protocol (HTTP)
```

Comment:
As far as I know `rpm` in SUSE verifies GPG signatures the same way as in RHEL or others.
The problem here is that download.opensuse.org server isn't accessible by HTTPS.

I think here we need to obtain and verify GPG key first (using hash), and after that configure the repo "manually" using local key file.
Just need to wait until #791 will be merged.